aks/application_configuration/variables.tf

```diff
@@ -53,6 +41,18 @@ variable "config_variables_path" {
   description = "Path to load additional configuration variables from"
 }
 
+variable "secret_name" {
```

Comment:
Should this be key_vault_name instead?

Reply:
It's used as the `name` argument to the secrets module, so I wanted to make it clear that it's _not_ the full name of the key vault.

Reply:
It looks like a secret inside keyvault though instead of the keyvault itself
Maybe key_vault_short? key_vault_id?